docs/user-guide/configuration.md

```diff
@@ -0,0 +1,63 @@
+
+---
+title: Configuration
+---
+
+vSphere Docker Volume Service consists of two components the VIB and Docker plugin. The Docker plugin requires configuration which is described in this section
+
+The Docker volume plugin at the moment supports vSphere and Photon platforms. All the volume provisioning and management operations are supported on both the platforms. Both the platforms can be used to provision VMDK based volumes and can be attached to containers. 
```

Comment:
Tenancy supported for vSphere and for Photon are very different. If tenant model is being described anywhere then in this para please mention that there are differences which will be highlighted as applicable - tenancy model, admin CLI (only for vSphere), vsphere-only volume create options like attach type, access type, clone-from, etc.